components/processing/form_parser/deployment/versions.tf

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+terraform {
+  required_version = ">=1.5.7"
+
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = ">= 5.23.0"
+    }
+  }
+
+  provider_meta "google" {
+    module_name = "cloud-solutions/eks-docai"
```

Comment:
you need to add a version number in the end in order for it to be picked up by the tracking, for instance 
`cloud-solutions/eks-docai-v1.2`

Reply:
@anuradha-bajpai-google - Can you make the suggested changes and submit a PR against version1.2?